plugins/twitter.py

```diff
@@ -55,7 +57,7 @@ def twitter_url(match):
 
     time = timeformat.time_since(tweet.created_at, datetime.utcnow())
 
-    return "{}@\x02{}\x02 ({}): {} ({} ago)".format(prefix, user.screen_name, user.name, text, time)
+    return "{}@\x02{}\x02 ({}): {} ({} ago)".format(prefix, user.screen_name, user.name, html.unescape(text), time)
```

Comment:
Since this formatting logic is duplicated in the `twitter()` function, perhaps it should be moved to it's own function.
Something like:
```python
def format_tweet(tweet):
    user = tweet.user
    text = " ".join(tweet.text.split())

    if user.verified:
        prefix = "\u2713"
    else:
        prefix = ""

    time = timeformat.time_since(tweet.created_at, datetime.utcnow())
    return "{}@\x02{}\x02 ({}): {} ({} ago)".format(prefix, user.screen_name, user.name, html.unescape(text), time)
```